tests/common/snappi_tests/common_helpers.py

```diff
@@ -821,6 +823,42 @@ def get_pfc_frame_count(duthost, port, priority, is_tx=False):
     return int(pause_frame_count.replace(',', ''))
 
 
+def get_tx_frame_count(duthost, port):
+    """
+    Get the Tx_OK and Tx_DRP frame count for a given port ex. Ethernet4 from SONiC CLI
+    Args:
+        duthost (Ansible host instance): device under test
+        port (str): port name ex. Ethernet4
+    Returns:
+        tx_frame_count (int): Tx frame count
+    """
+    raw_out = duthost.shell("portstat -ji {}".format(port))['stdout']
+    raw_out_stripped = re.sub(r'^.*?\n', '', raw_out, count=1)
+    raw_json = json.loads(raw_out_stripped)
+    tx_ok_frame_count = raw_json[port]["TX_OK"]
+    tx_drp_frame_count = raw_json[port]["TX_DRP"]
+
+    return int(tx_ok_frame_count.replace(',', '')), int(tx_drp_frame_count.replace(',', ''))
+
+
+def get_rx_frame_count(duthost, port):
```

Comment:
get_tx_frame_count and get_rx_frame_count are actually very similar. Can you please use a unified function to provide various TX/RX counters?